### DISCUSSION TITLE
Verification Issues

### DISCUSSION THREAD

**Discussion by notscar** [community/community#61089] [GitHub Education]
### Hi everyone,

I've reapplied plenty of times, now finally they approved the documentation but it now says im not near the campus, which i am, it's just that both my Home IP locates me in a different city (big city), and my cellular data ip locates me even more far...

i've opened a ticket over a week ago #2247563 (Github General Support) and #2251827 (Github Education, as suggested by a refusal reason)  but still did not get a response.

**Comment by samus-aran** [comment]
Hi @NotScar 

Being Far from Campus is taken into consideration so ensure your are selecting the correct school, avoid selecting similarly named schools. You should have information on your evidence that matches the selected school you are attending.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on.

**Reply by notscar** [reply]
I made sure i selected the corrected school, and the address it's matching, while i'd like to verify my accademic status with the school issued email, the GCloud organization rules don't allow students to receive or send emails, from emails that are not in the organization, so therefore it's impossible for me to receive the github confermation email, i've tried to make it forward and everything but the rules are set in place correctly and there is zero way to use that email until they allow us from the GCloud platform (next year probably), and the only other school document that i have other than the one i'm trying to verify with it's just a digital letter pdf, that indicates i've passed this academic year but does not specify the fact i'm going to be there for the next year, it just shows my grades, the schools information and this accademic year, i will try to submit this, but i don't think it's going to get me far.

**Reply by notscar** [reply]
@samus-aran I've now sent the certificate as i said in the previous post, and it was refused because i didn't use the academic mail address, i need to upload a different document. I've verified myself on SheerID for spotify student benefits in less than 3 hours with the same document, im not sure what to do anymore and both my tickets aren't getting any response

**Comment by notscar** [comment]
2 Weeks went by, no response in the ticket, and github told me my PRO is about to expire lol

**Comment by samus-aran** [comment]
Hi @01a12 

With each application you receive rejection reasons relevant to the last application made. Either copy & paste the latest rejection reasons or provide a screenshot so we can give you a nudge in the right direction.

**Reply by notscar** [reply]
Hey @samus-aran , i didn't submit any other application since i was waiting response from support, since it was pointed out to contact them, in order to sort out the fact that it said i wasn't near the school, in the refusal message, but i have resubmitted again a new one today and this is what i got from it


Attendees of this school must verify and use their academic email address.


Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.


1 - As i've pointed out we can't use emails since the organization GSuite made by the school has a policy that prevents inbound and outbound emails from emails that are not in the organization, unless you're a teacher or last year student, so i can't do that at all, and i wasn't required to do that last year
2 - It does include a date which is on top right and indicates the current accademic year 2022/23

**Comment by notscar** [comment]
Now, all the others notices are gone and i just have this, that tells me to verify with the email, which of course i can't, and yet no response from support after 3 weeks 
![image](https://github.com/community/community/assets/71942531/fcc4493e-7a6e-4819-970d-9b948aa01eed)

**Comment by Awrsha** [comment]
This does sound frustrating. A few things I would recommend:

Be patient - since you've already opened tickets, GitHub support is likely reviewing your case. But they may have a large volume of requests, so it can take time for them to respond.

Follow up on the tickets - after a week or so, you can politely follow up in the ticket to check on the status and remind them of your situation. Provide any additional details or clarification they might need.

Explain your situation clearly - in the ticket or any follow up, be sure to mention that:

You are a student near the campus, but your IP addresses locate you in a different city due to using a home internet connection or cellular data.

This is a technical issue, not an accuracy issue - you are physically located near the campus as required.

Requiring you to use campus WiFi would be impossible since GitHub is blocked on many campus networks.

Ask them how you can prove your student status and campus proximity to their satisfaction.

Escalate the ticket internally - if you still don't receive a response after following up, you can ask in the ticket if there is a way to escalate the issue to a manager or higher-level support team. Sometimes that can get things moving.

Remain polite and patient - I know this is frustrating, but staying polite and professional in your communications will help you get a more positive response from GitHub support.

**Reply by notscar** [reply]
Hey, thanks for reassuring, it defintly is frustrating because i remember the previous 2 years in less than 2 days i was verified correctly, i now have only an issue with not being able to use the email to verify my self, as i've written before because the gmail organization policy doesn't allow inbound or outbound emails from externals, and i was told to make a ticket for it, if it was wrong, and indeed i did 3 weeks back, but to me it makes no sense that such a big company backed up by such a big corporation does not have enough personnel to sustain the amount of tickets, i personally think that using AI to verify such a different variety of documents, is stupid, and not having a way to put into manual verification is even more stupid, essentially a flawed system.

**Comment by notscar** [comment]
Now another message

![image](https://github.com/community/community/assets/71942531/b4a26c51-eddb-487c-96c2-6bb984cad1eb)
In a couple days i'll hit the unlock all the denied message achivement

**Reply by notscar** [reply]
Im not sure why, after changing the school, (there was a duplicate or something like that) from the dropdown menu in the first form, and uploading the same exact document it was insta accepted
